var/spack/repos/builtin/packages/bcftools/package.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
Fixup the copyright headers

Reply:
OOPS!  We *did* increment the year recently, didn't we?! ;)